README.md

```diff
@@ -908,6 +908,26 @@ word-break:break-word
      </td>
      <td>Read</td>
```

Comment:
`README.md` should not be edited, just `README-template.md`

Reply:
Fixed